_jekyll/includes/2-base/navigation.html

```diff
@@ -37,6 +37,7 @@
         <ul class="links">
           <li><a href="{{ '/learning/nature-of-code' | relative_url }}">Nature of Code</a></li>
           <li><a href="{{ '/learning/ml5' | relative_url }}">ml5.js Beginners Guide</a></li>
+          <li><a href="{{ '/learning/discord-bot' | relative_url }}">Discord Bot</a></li>
```

Comment:
Would it make sense to call this just "bots" and ultimately it would hold both twitter bot and discord bot tutorials?

Reply:
Moved to be under a bots folder and then a discord subdirectory. Kept the link as Discord Bot but can be updated when we add twitter bot videos.